configure.ac

```
@@ -559,8 +559,7 @@ case $host in
        AC_MSG_ERROR("windres not found")
      fi
 
-     CPPFLAGS="$CPPFLAGS -D_MT -DWIN32 -D_WINDOWS -DBOOST_THREAD_USE_LIB"
-     LEVELDB_TARGET_FLAGS="-DOS_WINDOWS"
+     CPPFLAGS="$CPPFLAGS -D_MT -DWIN32 -D_WINDOWS -DBOOST_THREAD_USE_LIB -D_WIN32_WINNT=0x0601"
```

29283b67a82975ef195e43af42bae6cbe293356d: CPPFLAGS change is from 14922 which looks unrelated
```suggestion
     CPPFLAGS="$CPPFLAGS -D_MT -DWIN32 -D_WINDOWS -DBOOST_THREAD_USE_LIB"
```

---

Chalked it up to change in LevelDB's build system so ported it over

---

https://github.com/UdjinM6/dash/commit/453b11274da379da5a653f1378cfac059f8b747e -> https://gitlab.com/UdjinM6/dash/-/pipelines/336015433 I do not see any issues, CI is happy